src/app/clusters/bindings/PendingNotificationMap.cpp

```diff
@@ -98,6 +102,10 @@ void PendingNotificationMap::RemoveEntry(uint8_t bindingEntryId)
             mPendingContexts[newEntryCount]       = mPendingContexts[i];
             newEntryCount++;
         }
+        else if (mPendingContexts[i] != nullptr)
```

Comment:
Again, when can this be null?  As far as I can tell AddPendingNotification always passes in a non-null context....

Reply:
Answered in the previous comment